AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb

```diff
@@ -254,16 +239,15 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "ax.set_xlim([0.5, 4.5])\n",
     "ax.set_ylim([-2, 8])\n",
-    "ax.set_title('An Example Axes')\n",
-    "ax.set_ylabel('Y-Axis')\n",
-    "ax.set_xlabel('X-Axis')"
+    "ax.set_title('An Diferent Example Axes Tile')\n",
+    "ax.set_ylabel('Y-Axis (changed)')\n",
+    "ax.set_xlabel('X-Axis (changed)')\n",
+    "fig.canvas.draw_idle()"
```

Comment:
what's the point of this draw_idle(). I really don't want this introduced so early.